Docs/user_guide/modules/decimation.md

```diff
@@ -0,0 +1,33 @@
+# Decimation
+
+## Overview
+
+This module is used for performing a topology-preserving reduction of surface triangles.
+
+## Methods
+
+It implements the following methods:
+
+| Method | Description | Supported Format |
+|--------|-------------|------------------|
+| FastQuadric | Uses [Sven Forstmann's method][Sven-Forstmann] | `obj` |
+| Quadric | Uses [vtkQuadricDecimation][vtkQuadricDecimation] based on the work of Garland and Heckbert who first presented the quadric error measure at Siggraph '97 "Surface Simplification Using Quadric Error Metrics" | `obj`, `vtp` |
+| DecimatePro | Uses [vtkDecimatePro][vtkDecimatePro] implementing an approach similar to the algorithm originally described in "Decimation of Triangle Meshes", Proc Siggraph `92 | `obj`, `vtp` |
+
+[Sven-Forstmann]: https://github.com/sp4cerat/Fast-Quadric-Mesh-Simplification
+[vtkQuadricDecimation]: https://vtk.org/doc/nightly/html/classvtkQuadricDecimation.html#details
+[vtkDecimatePro]: https://vtk.org/doc/nightly/html/classvtkDecimatePro.html#details
+
+Notes:
+
+* Quadric filters provide much better shaped triangles, especially when large reduction ratio is requested.
+
+## Contributors
+
+Authors:
+- Beatriz Paniagua (UNC)
```

Comment:
@bpaniagua  Back in 2012, you migrated the code from NeuroLib (see https://github.com/NIRALUser/SPHARM-PDM/commit/429f7a0a45a8527ef3b78238d83124790734b1ca). Do you happen to recall who contributed to the project ?  (I wasn't able to checkout the corresponding CVS project based on [these instructions](https://web.archive.org/web/20130301124142/http://www.ia.unc.edu/dev/tutorials/Developer/neurolib-linux.htm))